README.md

```diff
@@ -84,10 +101,10 @@ In order to build all the examples from this repository the following dependenci
     -DWITH_HPX=ON -DHPX_DIR=[1]
     ```
     where [1] is a path to the subdirectory [...]/lib/cmake/HPX/ of the HPX build directory
-3. [Boost](https://www.boost.org/)
+1. [Boost](https://www.boost.org/)
 1. [QT5](http://doc.qt.io/qt-5/gettingstarted.html)
 
-### Building examples
+### Building examples (this repository)
```

Comment:
the whole readme is about this repository, therefore the following can be removed

> (this repository)